docs/components/gripper/softrobotics.md

```diff
@@ -85,8 +86,8 @@ The following attributes are available for `softrobotics` grippers:
 
 Name | Inclusion | Type | Description
 ---- | --------- | ---- | -----------
-`board` | **Required** | string | The name of the [board](../../board/) to which your gripper [control unit](https://www.softroboticsinc.com/uploads/2020/05/Tech_Sheet_coDrive_Control_Unit_-__TS-200210_Rev_B.pdf) is wired.
-`open` | **Required** | string | The pin number of the board pin wired to the open pin (D1) on the gripper controller.
-`close` | **Required** | string | The pin number of the board pin wired to the close pin (D2) on the gripper controller.
-`power` | **Required** | string | The pin number of the board pin wired to the enable pin (D3) on the gripper controller.
-`analog_reader` | **Required** | string | Needs to be called `"psi"`, and you need to configure an [analog](../../board/#analogs) named `"psi"` on your board.
+`board` | **Required** | string | The `name` of the [board](../../board/) to which your gripper [control unit](https://www.softroboticsinc.com/uploads/2020/05/Tech_Sheet_coDrive_Control_Unit_-__TS-200210_Rev_B.pdf) is wired.
+`open` | **Required** | string | The {{< glossary_tooltip term_id="pin-number" text="pin number" >}} of the board pin wired to the open pin (D1) on the gripper controller.
+`close` | **Required** | string | The {{< glossary_tooltip term_id="pin-number" text="pin number" >}} of the board pin wired to the close pin (D2) on the gripper controller.
+`power` | **Required** | string | The {{< glossary_tooltip term_id="pin-number" text="pin number" >}} of the board pin wired to the enable pin (D3) on the gripper controller.
+`analog_reader` | **Required** | string | Must be called `"psi"`. You must [configure an analog](../../board/#analogs) with the `"name"` of `"psi"` on your board.

```

Comment:
```suggestion
`analog_reader` | **Required** | string | Must be called `"psi"`. You must [configure an analog](../../board/#analogs) on your board and name it `"psi"`.
```
I think this might be even more clear? I could conceive of someone thinking that "configure it with the name of psi" means configure it with the same name that some thing called "psi" has.